src/frontend/js/utils/api/lms/openedx-hawthorn.spec.ts

```diff
@@ -143,6 +152,12 @@ describe('OpenEdX Hawthorn API', () => {
         );
 
         expect(response).toBeFalsy();
+        expect(mockHandle).toHaveBeenCalledWith(
+          new Error('[GET - Enrollment] > 500 - Internal Server Error'),
+        );
```

Comment:
This is a false positive. This assertion is true only because the mock is not restore before each tests. So in the call stack there is still the error caught in the previous test.

In the `beforeEach` function, you have to add `mockHandle.mockRestore`.